tests/test_cognitoidp/test_cognitoidp.py

```diff
@@ -1198,7 +1198,7 @@ def test_admin_delete_user():
     caught.should.be.true
 
 
-def authentication_flow(conn):
+def authentication_flow(conn, auth_flow):
```

Comment:
Looks like there are other tests that use this method, so we can probably just default the `auth_flow` param to `"ADMIN_NO_SRP_AUTH"`

Reply:
Thanks. I've been trying to run the tests locally but getting all sorts of errors I think due to python3 and installing the necessary project dependencies 😢
Will have another look today